components/dropbox/sources/new-folder/new-folder.mjs

```diff
@@ -0,0 +1,54 @@
+import dropbox from "../../dropbox.app.mjs";
+
+export default {
+  key: "dropbox-new-folder",
```

Comment:
If you create a `common.mjs` file in sources, you can remove `import dropbox`, and include common:
```
export default {
   ...common,
   key: "dropbox-new-folder",
```

Then you can remove the `props`, `hooks`, and `methods` sections from this action.